src/classes/CRLP_RollupProcessor.cls

```diff
@@ -499,8 +499,20 @@ public inherited sharing class CRLP_RollupProcessor {
         SObject updatedRecord = handler.getPopulatedSObject();
 
         // Determine if the updated SObject record is different than the parent.
-        // If not, set it to null
-        if (CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs) == false) {
+        Boolean needsUpdate = CRLP_Rollup_SVC.resultsNeedUpdate(parent, updatedRecord, rollupDefs);
+
+        // Set skew mode flag on skew mode records not currently set
+        String qualifiedSkewField = UTIL_Namespace.StrTokenNSPrefix('CustomizableRollups_UseSkewMode__c');
+        if (rollupType != CRLP_RollupProcessingOptions.RollupType.GAU
```

Comment:
@npsp-reedestockton  The only other thing to check for here is that the job is running in SkewMode. I think that's defined in the `mode` var.

Reply:
Thanks!  I lost track of the fact that I wasn't in a skew mode job for a second there...